### PR TITLE
update advises

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

thoth-station/thoth-application#1655

## This introduces a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

File: thoth/common/openshift.py [hunk 1]
```diff
@@ -1404,6 +1405,9 @@ def schedule_adviser(
         if count is not None:
             template_parameters["THOTH_ADVISER_COUNT"] = str(count)
 
+        if from_package_update:
+            template_parameters["THOTH_ADVISER_DEPLOYMENT_PACKAGE_UPDATE"] = "1"
```
Comment (fridex) [inline thoth/common/openshift.py:1409]: Any reason not to put directly `THOTH_ADVISER_DEPLOYMENT_PACKAGE_UPDATE` - JSON describing the package to be updated?

Comment (KPostOffice) [reply thoth/common/openshift.py:1409]: ++, I realized this information will be present in Kebechet metadata. Should we just duplicate it here if the message type is package_release?

Comment (fridex) [reply thoth/common/openshift.py:1409]: Yes, I think so - adviser will need to be triggered with adjusted environment variables to run in the "update" mode. The function can accept an optional triplet (package name, package version, index url) that could be serialized to the env variable. The triplet can be supplied based on the message received.

**Comment by goern** [comment]
Can we merge this?

**Review by fridex — APPROVED** [review]
lgtm, thanks 👍🏻

**Pull request merged**